weblate_web/templates/about.html

```diff
@@ -44,6 +44,36 @@ <h2>{% trans "Weblate grows and the future is open to new opportunities." %}</h2
                 </div>
             </div>
 
+            <h2 class="section-title crypto">{% trans "Your Weblate Team" %}</h2>
+            <div class="weblate-team">
+                <div class="weblate-team-content">
+                   <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                     <div class="content-p">
+                       <p>MICHAL CIHAR</p>
+                       <br>
+                       <t>Master of the code & head of custom or large scale solutions.</t>
```

Comment:
Please make the text (and all others translatable).

Reply:
The "trans" block has been added to the texts.